node/service/src/client.rs

```diff
@@ -42,7 +42,7 @@ pub trait RuntimeApiCollection:
 	+ fp_rpc::EthereumRuntimeRPCApi<Block>
 	+ moonbeam_rpc_primitives_debug::DebugRuntimeApi<Block>
 	+ moonbeam_rpc_primitives_txpool::TxPoolRuntimeApi<Block>
-	+ nimbus_primitives::AuthorFilterAPI<Block, nimbus_primitives::NimbusId>
+	+ nimbus_primitives::NimbusApi<Block, nimbus_primitives::NimbusId>
```

Comment:
If we have trouble authoring in the first block after the upgrade, this name change could be the culprit. I don't know how runtime api's are identified but it might be by `extern "C"`. In any case, this is a good rename, so if we have trouble I suggest we work around it by using the same force-authoring-on-first-block-after-upgrade trick as before.